docs.kosli.com/content/integrations/sonar.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Sonar
+bookCollapseSection: false
+weight: 340
+---
+# Sonar in Kosli
+
+The results of SonarCloud and SonarQube scans can be tracked in [Kosli trails](/getting_started/trails/).
```

Comment:
before diving into the how-to, it can be useful to have an overview of how this integration works and what's need for it to work.